### ISSUE TITLE
Documentation has wrong format online

### ISSUE THREAD

**Issue by zhangysh1995** [jepsen-io/jepsen#505]
Hi, I found that some parts of the documentation for [jepsen.generator](http://jepsen-io.github.io/jepsen/jepsen.generator.html)  has a wrong format. 

For example, in the source code there is:
````
  - The generator may return an operation, in which case:
    - If its time is in the past, we can evaluate it now
    - If its time is in the future, we wait until either:
      - The time arrives
      - Circumstances change (e.g. we update the generator)
````
But online it is:

> *  The generator may return an operation, in which case:
> *  If its time is in the past, we can evaluate it now
> *  If its time is in the future, we wait until either:
>    *  The time arrives
>    *  Circumstances change (e.g. we update the generator)

Similarly, the example code is not in the correct order:
````
  (op gen test context) => [op gen']      ; known op
                           [:pending gen] ; unsure
                           nil            ; exhausted
````

> (op gen test context) => [op gen’] ; known op [:pending gen] ; unsure nil ; exhausted

I want to know whether I can help to improve it?

**Comment by aphyr** [comment]
Yeah, this is probably a codox markdown thing--I'm usually looking at the source rather than the online docs. You're welcome to fix it if you like!
